dist/types/store/resource-viewer/index.d.ts

```diff
@@ -0,0 +1,18 @@
+import { ActionTree, GetterTree, Module, MutationTree } from 'vuex';
+import { IRootState } from '../../store';
+import { IBlock } from '@floip/flow-runner';
+export interface IResourceViewerState {
+    /**
+     * Block uuids to hide,
+     * Will be used for resource-viewer component, so the consumer repo can set a custom filters and override this state if needed
+     *
+     * By default, it's empty, meaning there is no filter at all
+     */
+    blockUuidsToHide: IBlock['uuid'][];
```

Comment:
@bulatgab , I'm curious why do we need those changes in current PR ?

Reply:
I think it is not new, it was just missing.
Probably after creating the `index.ts` source file and running `yarn build-bundle` this `.d.ts` was generated but not git-added.
I think we could maybe append `git add dist/` to the `"build-bundle"` script.

Reply:
Got you, I'd didn't notice it was in `dist/`. thanks